docs/source/index.rst

```diff
@@ -0,0 +1,11 @@
+Jax AI Stack
+=====================================
```

Comment:
The length of the underline should match the length of the test (I believe sphinx generates warnings if it doesn't)

Reply:
Updated